requirements.txt

```diff
@@ -1,10 +1,10 @@
-bayesian-changepoint-detection>=0.2.dev1
+git+https://github.com/hildensia/bayesian_changepoint_detection@2dd95f5c1d028116899a842ccb3baa173f9d5be9
 changepy>=0.3.1
 hyperopt>=0.1.2
 numpy>=1.17.5
 pandas>=0.25.3
 pykalman>=0.9.5
-scipy<=1.2.3
-statsmodels>=0.10.2,<=0.12.2
+scipy>=1.7.3
```

Comment:
This are the most recent versions of those packages, have we checked if our code is compatible with the the last stable version?

Reply:
I'm less worries about scipy since it's a minor version bump, but since we're doing a major bump for statsmodels (due to zerover) yes we need to validate if there are other api changes

Reply:
Yes we have checked the compatibility with the latest versions of both scipy and statsmodels

Reply:
My questions was more around providing a min version which is not the most recent version for scipy. So, have we tested scipy with any other version >1.2? 
Or we should at-least change this to >=1.7.0 if thats supported.

Reply:
I have updated the requirements